app/assets/src/components/views/discovery/DiscoveryView.jsx

```diff
@@ -800,88 +757,72 @@ class DiscoveryView extends React.Component {
   };
 
   refreshMapPreviewedSamples = async () => {
-    const { domain } = this.props;
-    const {
-      mapLocationData,
-      mapPreviewedLocationId,
-      projectId,
-      search,
-    } = this.state;
-
-    if (!mapPreviewedLocationId) return;
+    const { mapLocationData, mapPreviewedLocationId } = this.state;
 
-    if (!mapLocationData[mapPreviewedLocationId]) {
+    if (!mapPreviewedLocationId || !mapLocationData[mapPreviewedLocationId]) {
```

Comment:
When I click the tooltip title to preview a location, I don't see the samples list actually load

![Screen Shot 2019-09-19 at 5 41 40 PM](https://user-images.githubusercontent.com/5652739/65290606-d2882280-db04-11e9-931c-44d8a9c001e2.png)

Reply:
Should be fixed.